code-challenges.test.js

```diff
@@ -22,10 +22,55 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisible by three"
 
+describe("divThree", () => {
+  let num1 = 15
+  let num2 = 0
+  let num3 = -7
+  it("takes a number as an argument and decides if the number is evenly divisble by three or not", () => {
+    // Expect each number to deliver the expected output
+
+    expect(divThree(15)).toEqual("15 is divisable by 3")
+    expect(divThree(0)).toEqual("0 is not divisable by 3")
+    expect(divThree(-7)).toEqual("-7 is not divisable by 3")
+  })
+})
+// Test fail. divThree is not defined.
+
 
 
 // b) Create the function that makes the test pass.
 
+// Create a function, capable of testing each argument. Use the parameter of num.
+const divThree = (num) => {
+  // create an if/else statement to map out the different out comes. 
+  // If/else should decide for 0, else if num is div by 3 without a remainder. Create else for every other outcome.
+  if (num === 0) {
+    return `${num} is not divisable by 3`
+  } else if (num % 3 === 0) {
+    return `${num} is divisable by 3`
+  } else {
+    return `${num} is not divisable by 3`
+  }
+}
+// log function with argument.
+// console.log(divThree(num1));
+// returns ---> 15 is divisable by 3
+// console.log(divThree(num2));
+// returns ---> 0 is not divisable by 3
+// console.log(divThree(num3));
+// returns ---> -7 is not divisable by 3
+
+// FINAL
+// const divThree = (num)=>{
+//   if (num === 0) {
+//     return `${num} is not divisable by 3`
+//   } else if (num % 3 === 0){
+//     return `${num} is divisable by 3`
+//   } else {
+//     return `${num} is not divisable by 3`
+//   }
+// }
+// console.log(divThree(num1));
```

Comment:
Looks great. Just for reference, whatever code is currently active (aka not commented) is what I am going to review. All the code looks good, just noting your "FINAL" function that is currently not able to run due to being commented out.